vec-redis/setup-redis.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Usage: setup-redis.sh
+set -eEuo pipefail
+
+tmpdir="${PREM_APPDIR:-.}/redis-$(uuidgen)"
```

Comment:
wait you needed `uuidgen` instead of `uuid`? Maybe also update `cht-petals/setup-petals.sh` then?

Reply:
yeahh I actually forgot to make this change there too, will make it now

Reply:
https://github.com/premAI-io/prem-services/pull/133

Reply:
```suggestion
tmpdir="${PREM_APPDIR:-.}/redis-$(uuidgen 2>/dev/null || uuid)"
```